redbox-core/redbox/models/file.py

```diff
@@ -22,12 +22,21 @@ class ChunkMetadata(BaseModel):
     """
 
     uuid: UUID = Field(default_factory=uuid4)
-    index: int
-    file_name: str
-    page_number: int | None = None
+    index: int = 0  # The order of this chunk in the original resource
     created_datetime: datetime.datetime = datetime.datetime.now(datetime.UTC)
-    token_count: int
     chunk_resolution: ChunkResolution = ChunkResolution.normal
+    creator_type: str
```

Comment:
```suggestion
    creator_type: Literal["..."]
```
to match the equivalent in django ?